commit f72151fb6820e90ce12a15e2768aa41150c5186d
Author: Hin-Tak Leung <hintak.leung@gmail.com>
Date:   Tue Jun 16 03:15:56 2009 +0100

    zd1211rw: adding 083a:e503 as a ZD1211B device
    
    Hans Pontar reported success on the sourceforge zd1211-devs mailing list.
    The device is branded "Arcor Easy Stick A 50 WLAN" (device manufactured
    by SMC for a German ISP - SMC model name: WN4501H-LF-IR). General
    information and Windows driver are available under (German only):
    
    http://www.arcor.de/hilfe/neu/index.php?sid=&aktion=anzeigen&rubrik=004018140&id=487
    
    Device details:
    
    USB-IDs: Vendor: 0x083A Device: 0xE503
    Chip ID: zd1211b chip 083a:e503 v4810 high 00-1d-19 AL2230S_RF pa0 g--N-
    
    Signed-off-by: Hin-Tak Leung <htl10@users.sourceforge.net>
    Tested-by: Hans Pontar <pontar@gmx.de>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

diff --git a/drivers/net/wireless/zd1211rw/zd_usb.c b/drivers/net/wireless/zd1211rw/zd_usb.c
index f0e5e94..14a19ba 100644
--- a/drivers/net/wireless/zd1211rw/zd_usb.c
+++ b/drivers/net/wireless/zd1211rw/zd_usb.c
@@ -67,6 +67,7 @@ static struct usb_device_id usb_ids[] = {
 	{ USB_DEVICE(0x079b, 0x0062), .driver_info = DEVICE_ZD1211B },
 	{ USB_DEVICE(0x1582, 0x6003), .driver_info = DEVICE_ZD1211B },
 	{ USB_DEVICE(0x050d, 0x705c), .driver_info = DEVICE_ZD1211B },
+	{ USB_DEVICE(0x083a, 0xe503), .driver_info = DEVICE_ZD1211B },
 	{ USB_DEVICE(0x083a, 0xe506), .driver_info = DEVICE_ZD1211B },
 	{ USB_DEVICE(0x083a, 0x4505), .driver_info = DEVICE_ZD1211B },
 	{ USB_DEVICE(0x0471, 0x1236), .driver_info = DEVICE_ZD1211B },
